spring-integration-core/src/test/java/org/springframework/integration/support/management/graph/IntegrationGraphServerTests.java

```diff
@@ -59,6 +61,7 @@
  *
  */
 @RunWith(SpringJUnit4ClassRunner.class)
+@DirtiesContext
 public class IntegrationGraphServerTests {
```

Comment:
Interesting... I have just noticed.
This test class is without `@ContextConfiguration`.
That is one more good thing from SF-4.3 : 
```
AbstractTestContextBootstrapper
...
if (MetaAnnotationUtils.findAnnotationDescriptorForTypes(
		testClass, ContextConfiguration.class, ContextHierarchy.class) == null) {
	return buildDefaultMergedContextConfiguration(testClass, cacheAwareContextLoaderDelegate);
}
...
/**
 * @since 4.3
 */
private MergedContextConfiguration buildDefaultMergedContextConfiguration(Class<?> testClass,
			CacheAwareContextLoaderDelegate cacheAwareContextLoaderDelegate) {

```
:smile: 

Merging, BTW.
Nothing to complain about.